### PR TITLE
Document account keys

### PR DESCRIPTION
**Pull request by geigerzaehler**
Document what we use for account keys and how to create them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,32 @@ You’ll need to build the client with Rust Nightly.
 To build and view the client documentation run `./scripts/build-client-docs
 --open`.
 
+
+Account Keys
+------------
+
+We use Ed25519 keys for accounts. Key management functionality is provided by
```
Comment (xla) [inline README.md]: Does `management` imply anything beyond standardised creation of keys?

File: README.md [hunk 2]
```diff
@@ -16,6 +16,32 @@ You’ll need to build the client with Rust Nightly.
...
+the `radicle_registry_client::ed25519` module. To use this module you will
+likely need to import the `radicle_registry_client::CryptoPair` and
```
Comment (xla) [inline README.md]: Is there a scenario where these traits don't need to be imported for the `ed25519` module to be usable?

File: README.md [hunk 3]
```diff
@@ -16,6 +16,32 @@ You’ll need to build the client with Rust Nightly.
...
+`radicle_registry_client::CryptoPublic` traits.
+
+You can create key pairs using [`CryptoPair::generate()`][api-pair-generate]. To
+create keys from human readable strings, use
+```rust
+let alice = radicle_registry_client::ed25519::Pair::from_string("//Alice", None);
+```
+See [`CryptoPair::from_string`][api-pair-from-string].
+
+The `ed25519` module and the crypto traits are re-exports from
```
Comment (xla) [inline README.md]: Does this relate to the comment above?

File: README.md (hunk 1 above)
Comment (geigerzaehler) [reply README.md]: You’ll also need to serialize keys probably. But “management” sounds to broad I changed it to “key creation and handling functionality”.

File: README.md (hunk 2 above)
Comment (geigerzaehler) [reply README.md]: If you only use the types, say `ed25519::Public` then you don’t need the traits.

File: README.md (hunk 3 above)
Comment (geigerzaehler) [reply README.md]: Not sure what you mean

**Pull request merged**